src/Lit/LitElement.fs

```diff
@@ -256,9 +256,43 @@ type LitElementAttribute(name: string) =
 
         box(fun () -> failwith $"{name} is not immediately callable, it must be created in HTML") :?> _
 
-type LitElement =
-    static member inline init() =
-        jsThis<ILitElementInit<unit>>.init(fun _ -> ())
-
-    static member inline init(initFn: LitConfig<'Props> -> unit) =
-        jsThis<ILitElementInit<'Props>>.init(initFn)
+[<AutoOpen>]
+module LitElementExt =
+    open Browser
+    open Browser.Types
+
+    // TODO: Not sure why we don't have this constructor in Fable.Browser, we should add it
+    // CustomEvent constructor is also transpiled incorrectly, not sure why
+    [<Emit("new Event($0, $1)")>]
+    let private createEvent (name: string) (opts: EventInit): Event = jsNative
```

Comment:
These are the constructors I was referring as helpers in #14 glad/unglad you noticed them